eessi/testsuite/eessi_mixin.py

```diff
@@ -206,6 +206,18 @@ def EESSI_mixin_log_runtime_info(self):
             get_full_modpath = f'echo "FULL_MODULEPATH: $(module --location show {self.module_name} 2>&1)"'
             self.postrun_cmds.append(get_full_modpath)
 
+    @run_after('setup')
+    def EESSI_mixin_skip_offline_partitions(self):
+        """
+        Skip tests when download is required to run the test.
+        And there is no workaround the run to the test without downloading the files.
+        """
+        if self.module_name.split('/')[0] in ['GROMACS', 'MetalWalls', 'QuantumESPRESSO']:
```

Comment:
it's better to let the tests themselves advertise that they require internet.
i've made a PR to your PR for this in https://github.com/laraPPr/EESSI-test-suite/pull/1